packages/calcite-components/src/tests/commonTests/themed.ts

```diff
@@ -374,7 +374,7 @@ async function assertThemedProps(page: E2EPage, options: TestTarget): Promise<vo
   const isLinearGradientUnderlineToken = token.includes("link-underline-color") && targetProp === "backgroundImage";
 
   if (isFakeBorderColorToken || isLinearGradientUnderlineToken) {
-    expect(getStyleString(token, targetProp, styles[targetProp])).toMatch(expectedValue);
```

Comment:
We should keep this for testing maintenance. Removing `getStyleString` would remove helpful details from the test failure messages (see https://github.com/Esri/calcite-design-system/pull/9481).

Reply:
thanks for the catch I think this was a merge mistake